test/src/test/java/hudson/tasks/ArtifactArchiverTest.java

```diff
@@ -58,17 +59,30 @@
 
 import org.junit.Rule;
 import org.junit.Test;
-import org.jvnet.hudson.test.BuildWatcher;
-import org.jvnet.hudson.test.Issue;
-import org.jvnet.hudson.test.JenkinsRule;
-import org.jvnet.hudson.test.TestBuilder;
+import org.jvnet.hudson.test.*;
```

Comment:
Better to avoid it though not a big deal for the test code

Reply:
Whoops, my IDE did that